### ISSUE TITLE
Front lights glows red with new render

### ISSUE THREAD

**Issue by Talkless** [leezer3/OpenBVE#446]
### Description

Since 1.7.1.2 with New Render, front lights of Class 323 25kV AC EMU glows red. Glows white if New Render is not selected.

On 1.7.1.3it glows red with both old and new render, though it seems New glows more (bigger red blob).

On Master (68915a193750be3c1d25f538cd56f721018c7ca5) it's the same as with 1.7.1.3 - always red.

See screenshots with 1.7.1.2 (old render - OK) and master (both renders):
![openbve_white](https://user-images.githubusercontent.com/292191/77467411-a4b5ca80-6e14-11ea-950d-a148b162f94b.png)
![openbve_red](https://user-images.githubusercontent.com/292191/77467417-a5e6f780-6e14-11ea-96db-1a3178b301d5.png)

### Reproduction

Download Cross-city track from https://railsimroutes.net/x-city_south/index.php with Class 323 train, open OpenBVE Night route, turn power on.

## Route

https://railsimroutes.net/x-city_south/index.php

## Train

https://railsimroutes.net/x-city_south/index.php

## Logs

N/A

### Related information 

* Debian Sid amd64

**Comment by leezer3** [comment]
https://vps.bvecornwall.co.uk/OpenBVE/Builds/OpenBVE-2020-03-26.zip

Should fix that.

The blob appears approximately the same to me, I think it's just the blending interacting a little differently.

**Comment by Talkless** [comment]
> I think it's just the blending interacting a little differently.

Yes probably, but latest master still glows in red for me...

**Comment by leezer3** [comment]
It builds them automatically (Everythjng other than the Windows installers is automatic) :)

One of the better usages of CI.

I'll take a closer look at the exact histogram and colors at some stage, but things are crazy in retail at the minute....
I also wonder if the fixed errata lighting issue is interacting here.

**Comment by leezer3** [comment]
We appear to have a mixture of things going on here.

I've discovered a small further lighting anomaly in the 3D cab ( https://github.com/leezer3/OpenBVE/commit/a8103d7302ffcb416bfd2acd5c566a1c6782b678 - The cab was being lit by pure white, not mid-grey), which makes the overall appearance of the panel / red light marginally darker. 

It's still not exactly the 'same' as produced by the old renderer, but that is caused by this: https://github.com/leezer3/OpenBVE/wiki/Errata#lighting-behaviour-with-the-same-texture-defined-for-both-daytime-and-nighttime
TLDR: When the panel is activated, the entire 3D model of the cab is switched, and neither of these use alternate textures for day / night, hence are affected by the above.

As of this, I *think* (and stand to be corrected :P ) that the cab is now correctly lit in pure openGL terms.

**Comment by Talkless** [comment]
Not sure @leezer3 why it this closed? It's "wontfix"?

**Comment by leezer3** [comment]
I thought master was OK now, other than some minor shading differences?

**Comment by Talkless** [comment]
> Yes probably, but latest master **still glows in red for me**...

I'll try again though.

**Comment by leezer3** [comment]
Confusing even myself here.....

Will take another look when home, I'm sure I reproduced and fixed....

**Comment by Talkless** [comment]
Just rebuilt, launcher says `1.7.206.741-mysername`, and I still get reddish glow on the Night track of X-City South once I turn power on.

**Comment by leezer3** [comment]
This works OK for me in latest master, seems to be the same as 1.4:

Startup procedure:
Reverser to F
Reverser to N
Reset AWS self-test (Insert)
Turn on master switch (2)
**Tail lights activate, red glow to the front**
De-activate tail lights (4)
Set head lights (5 either once or twice depending if you want low or full)

**Comment by Talkless** [comment]
> seems to be the same as 1.4 (both renderers are the same):

I had no issue until I started using new render, hence, this bug report :/ .

Anyways, it seems a feature, not a bug, I just have to disable tail lights! There where no tail lights before :) .

Thanks!
